src/docs-md/framework-integration/ember.md

```diff
@@ -0,0 +1,15 @@
+# Ember
```

Comment:
It doesn't seem like these files are being used, but I figured it didn't hurt to add the file here as well as adding to the other page.